cico_run_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+. cico_setup.sh
+
+build_image
+
+#IMAGE_NAME=$(make get-image-name) ./runtest.sh
```

Comment:
It would be better to add some tests :) At least some smoke tests that the container is running and not crashing immediately.

Reply:
Ya I understand. But I have raised this to make sure CICD is ready. Post that I can add test cases. I have to refactor codebase as well.